gprt/gprt.cpp

```diff
@@ -2223,30 +2259,37 @@ struct Context {
       instanceCreateInfo.ppEnabledExtensionNames = instanceExtensions.data();
     }
 
-    // The VK_LAYER_KHRONOS_validation contains all current validation functionality.
-    // Note that on Android this layer requires at least NDK r20
-    const char* validationLayerName = "VK_LAYER_KHRONOS_validation";
-    if (validation())
-    {
-      // Check if this layer is available at instance level
-      uint32_t instanceLayerCount;
-      vkEnumerateInstanceLayerProperties(&instanceLayerCount, nullptr);
-      std::vector<VkLayerProperties> instanceLayerProperties(instanceLayerCount);
-      vkEnumerateInstanceLayerProperties(&instanceLayerCount, instanceLayerProperties.data());
-      bool validationLayerPresent = false;
-      for (VkLayerProperties layer : instanceLayerProperties) {
-        if (strcmp(layer.layerName, validationLayerName) == 0) {
-          validationLayerPresent = true;
-          break;
-        }
-      }
-      if (validationLayerPresent) {
-        instanceCreateInfo.ppEnabledLayerNames = &validationLayerName;
-        instanceCreateInfo.enabledLayerCount = 1;
-      } else {
-        std::cerr << "Validation layer VK_LAYER_KHRONOS_validation not present, validation is disabled";
-      }
-    }
+    instanceCreateInfo.ppEnabledLayerNames = nullptr;
+    instanceCreateInfo.enabledLayerCount = 0;
+    // Nate 11/29/2022 - not sure why, but adding VK_LAYER_KHRONOS_validation causes
```

Comment:
Did some of this commented code get overlapped with profiling work?

Reply:
Okay, sounds good. Thanks for verifying!

Reply:
So, should we go ahead and remove these commented lines then? Sounds like yes.

Reply:
We can, yep.